front_end/content.py

```diff
@@ -252,11 +264,37 @@ def get_problem_statuses(self, user_id, course_id, assignment_id, show_hidden=Tr
                  ORDER BY p.title'''
         self.c.execute(sql,(str(assignment_id), str(user_id),))
         for row in self.c.fetchall():
-            problem_dict = {"id": row["problem_id"], "title": row["title"], "passed": row["passed"]}
+            problem_dict = {"id": row["problem_id"], "title": row["title"], "passed": row["passed"], "num_submissions": row["num_attempts"]}
             problem_statuses.append([row["problem_id"], problem_dict])
 
         return problem_statuses
 
+    def get_assignment_scores(self, course_id, assignment_id):
+        #gets all users who have submitted on a particular assignment and creates a list of their average scores for the assignment
```

Comment:
I know it's just a comment, but please use proper punctuation and capitalization.